### ISSUE TITLE
Uno + TX Speed

### ISSUE THREAD

**Issue by ahshah** [pouriap/TinyRF#10]
Hi, 
I'm using TinyRF on an UNO and when using the following snippet:

```
  const char* msg = "Hello!";
  sendMulti((byte*)msg, strlen(msg), 2);
```
I noticed that its taking upwards of 330ms to perform the transmission. Is that expected? When using the radiohead library, transmission seems to be much faster, just curious as to what might be the reason.

**Comment by pouriap** [comment]
Yes that is expected because this library was mainly intended to be used with an ATtiny as the transmitter which is usually used with the internal oscillator which is not very fast or accurate.

If you are using an UNO as transmitter you can increase the speed in `Settings.h` file. 
Change the line that says `#define TRF_BITRATE_1000` to `#define TRF_BITRATE_2000`. You should get almost double the speed with that.
